syftr/flows.py

```diff
@@ -114,9 +114,19 @@ def get_prompt(self, query) -> str:
         examples = self.get_examples(query)
         assert examples, "No examples found for few-shot prompting"
 
+        thinking = ""
+        use_reasoning = (
+            None
+            if self.params is None
+            else self.params.get("response_synthesizer_llm_use_reasoning", None)
```

Comment:
I think this should be an explicit attribute of the Flow class like all the other attributes (though it's fine for `self.use_reasoning` to be defaulted to None)

Reply:
Good point

Reply:
Done